tokio-console/src/config.rs

```diff
@@ -156,6 +188,44 @@ impl KnownWarnings {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub(crate) enum AllowedWarnings {
```

Comment:
nit: it would be great to add a short docstring to this enum and its options so that we know what they mean.

Reply:
Added.